src/DurableTask/TaskOrchestrationContext.cs

```diff
@@ -24,6 +25,11 @@ public abstract class TaskOrchestrationContext
     /// </summary>
     public abstract string InstanceId { get; }
 
+    /// <summary>
+    /// Gets the parent instance. Null if no parent orchestration.
+    /// </summary>
+    public abstract ParentInstance? Parent { get; }
```

Comment:
For breaking changes like this, there are two things we can consider:
1. We can just make the change and document it in the CHANGELOG.md file. It's expected that we'll make some breaking changes prior to releasing the "GA" version of the SDK. No need for a major version bump yet.
2. Instead of making this `abstract`, we could make it `virtual` with a default implementation that does something such as `throw new NotImplementedException()`.

Reply:
Went with option 1. Would rather capitalize on the preview state and do the right thing now.